core/src/main/java/com/datastax/oss/driver/api/core/config/DriverConfigLoader.java

```diff
@@ -34,6 +36,8 @@
  */
 public interface DriverConfigLoader extends AutoCloseable {
 
+  static final ClassLoader DRIVER_CLASS_LOADER = CqlSession.class.getClassLoader();
```

Comment:
Maybe add javadocs here and explain briefly that this is used to load classpath config resources and that most people should not care about it.